addons/common/functions/fnc_headBugFix.sqf

```diff
@@ -0,0 +1,25 @@
+/**
+ * fnc_headbugfix.sqf
+ * @Descr: Fixes animation issues that may get you stuck
+ * @Author: rocko
+ *
+ * @Arguments:
+ * @Return: nil
+ * @PublicAPI: true
+ */
+
+#include "script_component.hpp"
+if (player != vehicle player) exitWith {};
```

Comment:
Also it is lacking any kind of check for for example the unconscious state. It is possible to use fix headbug while unconscious, even though it's a valid state. I don't think that should be the case, even though most likely you will be moved back into the unconscious animation after.